src/fundamental/types-fundamental.h

```diff
@@ -9,13 +23,17 @@ typedef CHAR16  sd_char;
 typedef INTN    sd_int;
 typedef UINTN   size_t;
 
-#define true    TRUE
-#define false   FALSE
+#define sd_true  TRUE
```

Comment:
I don't this is necessary. <stdbool.h> is not used in sd-boot/gnu-efi. And if it ever gets introduced, gcc will complain about it.